.github/workflows/pre-merge-integration-test.yml

```diff
@@ -1,5 +1,8 @@
 name: Pre-merge integration test
 on:
+  schedule:
+    # Every weekday at 5am
+    - cron: '0 5 * * 1-5'
```

Comment:
we don't need a cron for pre-merge integration tests

Reply:
@gokhakan can you please comment? or can we remove it?

Reply:
Agreed.